templates/chat/turn-card.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="dnd5e2 chat-card turn-card">
+    {{#if actor}}
+
+    {{!-- Deltas --}}
+    {{#if deltas.length}}
+    <section class="deltas">
+        <h5 class="roboto-condensed-upper">{{ localize "DND5E.CHATMESSAGE.TURN.Recovery" }}</h5>
```

Comment:
Prefer a `<strong>` rather than a heading element since it's not really clear to me where such a heading exists in the page's hierarchy.

Reply:
I was working off the message header which is an `<h4>`.

Reply:
I think I ended up not changing that in v13 due to backwards compatibility concerns, but the message header should also probably be a `<strong>`.